packages/demo/src/modules/user/signals/createFirebaseUser.js

```diff
@@ -0,0 +1,27 @@
+import {set, state, input} from 'cerebral/operators'
+import createUser from '../actions/createUser'
+import firebaseInit from '../../app/signals/firebaseInit'
+
+const createFirebaseUser = [
+  createUser, {
+    success: [
+      set(state`user.$loggedIn`, true),
+      set(state`user.signIn.$email`, ''),
+      set(state`user.signIn.$password`, ''),
```

Comment:
I would use `user.$signIn.email` and `user.$signIn.password` since it's the whole signIn thing that is "local UI stuff".